<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rocket Ground</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121212;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="1,6">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="4">
         <widget class="QChartView" name="chart4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #121212;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QChartView" name="chart3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #121212;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QChartView" name="chart1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #121212;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QChartView" name="chart2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #121212;
color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0,0,0,0,0,0,0,0">
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>by Ethan Armstrong</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <layout class="QHBoxLayout" name="settings_frame">
          <item>
           <widget class="QPushButton" name="settings">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border-image: url(:/images/settings.png);
}

QPushButton:pressed
{
	border-image: url(:/images/settings_pressed.png);
}

QPushButton:hover:!pressed
{
   border-image: url(:/images/settings_hover.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QFrame" name="led_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>119</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="led2_txt">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="led1_txt">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignHCenter">
            <widget class="QWidget" name="led1" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignHCenter">
            <widget class="QWidget" name="led2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="buttons_frame">
          <item>
           <widget class="QPushButton" name="btn1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: 1px solid #999999;
   background-color: rgba(62, 96, 193, 0);
	border-radius: 5px;
}

QPushButton:pressed
{
	border-color: rgba(62, 96, 193, 255);
   background-color: rgba(62, 96, 193, 31);
}

QPushButton:hover:!pressed
{
   background-color: rgba(62, 96, 193, 11);
}</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: 1px solid #999999;
   background-color: rgba(62, 96, 193, 0);
	border-radius: 5px;
}

QPushButton:pressed
{
	border-color: rgba(62, 96, 193, 255);
   background-color: rgba(62, 96, 193, 31);
}

QPushButton:hover:!pressed
{
   background-color: rgba(62, 96, 193, 11);
}</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: 1px solid #999999;
   background-color: rgba(62, 96, 193, 0);
	border-radius: 5px;
}

QPushButton:pressed
{
	border-color: rgba(62, 96, 193, 255);
   background-color: rgba(62, 96, 193, 31);
}

QPushButton:hover:!pressed
{
   background-color: rgba(62, 96, 193, 11);
}</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: 1px solid #999999;
   background-color: rgba(62, 96, 193, 0);
	border-radius: 5px;
}

QPushButton:pressed
{
	border-color: rgba(62, 96, 193, 255);
   background-color: rgba(62, 96, 193, 31);
}

QPushButton:hover:!pressed
{
   background-color: rgba(62, 96, 193, 11);
}</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>300</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Rocket Grorund v1.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionChart1">
   <property name="text">
    <string>Chart1</string>
   </property>
  </action>
  <action name="actionChart2">
   <property name="text">
    <string>Chart2</string>
   </property>
  </action>
  <action name="actionChart3">
   <property name="text">
    <string>Chart3</string>
   </property>
  </action>
  <action name="actionChart1_2">
   <property name="text">
    <string>Chart1</string>
   </property>
  </action>
  <action name="actionChart2_2">
   <property name="text">
    <string>Chart2</string>
   </property>
  </action>
  <action name="actionChart3_2">
   <property name="text">
    <string>Chart3</string>
   </property>
  </action>
  <action name="actionChart4">
   <property name="text">
    <string>Chart4</string>
   </property>
  </action>
  <action name="actionButtons">
   <property name="text">
    <string>Buttons</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>settings</tabstop>
  <tabstop>chart1</tabstop>
  <tabstop>chart2</tabstop>
  <tabstop>chart3</tabstop>
  <tabstop>chart4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
